### PR TITLE
AOS-2410 change url

### PR DESCRIPTION
**Pull request by bjartek**

### CODE REVIEW

**Comment by bjartek** [comment]
How do we send in routeSuffix here? I am not sure I like the proposed solution. 

Note: I tried to refactor defaults code from filesystem into an separate AuroraConfigFile but I ran into lots of test failures and I do not feel it adds enough value to do something with it right now.

File: src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraConfigFile.kt [hunk 1]
```diff
@@ -16,7 +16,12 @@ data class AuroraConfigFile(val name: String, val contents: String, val override
 
     val asJsonNode: JsonNode by lazy {
         try {
-            jacksonYamlObjectMapper().readValue(contents, JsonNode::class.java)
+            val mapper = if (name.endsWith(".json")) {
```
Comment (bjartek) [inline src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraConfigFile.kt]: Hmm why is this code here? It is already commited in develop earlier is it not?

File: src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraRouteMapperV1.kt [hunk 2]
```diff
@@ -2,41 +2,55 @@ package no.skatteetaten.aurora.boober.mapper.v1
...
 
-        val simplified=auroraConfigFields.isSimplifiedConfig("route")
+        val simplified = auroraConfigFields.isSimplifiedConfig("route")
         if (simplified && auroraConfigFields.extract("route")) {
-            return listOf(Route(name = name))
+            return listOf(Route(objectName = name,
+                    name = auroraConfigFields.extract("routeDefaults/name"),
+                    affiliation = auroraConfigFields.extract("routeDefaults/affiliation"),
+                    env = auroraConfigFields.extract("routeDefaults/env"),
+                    seperator = auroraConfigFields.extract("routeDefaults/separator")))
```
Comment (oyvinder) [inline src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraRouteMapperV1.kt]: Burde det vært separat retur for route=false?

File: src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraDeploymentSpec.kt [hunk 3]
```diff
@@ -165,11 +151,17 @@ data class Probe(val path: String? = null, val port: Int, val delay: Int, val ti
 
 
 data class Route(
+        val objectName: String,
         val name: String,
-        val host: String? = null,
+        val affiliation: String,
+        val env: String,
+        val seperator: String,
```
Comment (oyvinder) [inline src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraDeploymentSpec.kt]: separator

**Pull request merged**